src/validation.js

```diff
@@ -0,0 +1,36 @@
+const CASES = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+const validate = (text, caseName) => {
+  const errors = [];
+
+  if (text === '') {
```

Comment:
```suggestion
  if (!text) {
```
You don't need to compare with an empty string, just check if exists